### ISSUE TITLE
Java Debug Stick block state change does not apply.

### ISSUE THREAD

**Issue by arima0k** [GeyserMC/Geyser#3125]
### Describe the bug

Changing block states with the debug stick, example if I disconnect fences from blocks they still apear connected for all bedrock players.

### To Reproduce

1. Put a fence next to a solid block.
2. Disconnect the side of the fence that is connected to que solid block with the debug stick in Java.
3. Connect to the server with bedrock and look if the fence is connected to the solid block.

### Expected behaviour

Block state synced with bedrock block states.

### Java
![image](https://user-images.githubusercontent.com/25037200/178133797-c6eb727f-f629-4022-a08d-988005ecbbe8.png)

### Bedrock
![image](https://user-images.githubusercontent.com/25037200/178133805-b41c041f-e9e0-4bfe-9083-b19ffb516cc9.png)

### Server Version and Plugins

Velocity 3.1.2-SNAPSHOT (git-e8bf6ab5-b161)
Paper version git-Paper-56 (MC: 1.19)
floodgate 2.2.0-SNAPSHOT (b70-6e2c19f) (velocity and backend)

### Geyser Version

Geyser version 2.0.4-SNAPSHOT (git-master-2ce7ff8)

### Minecraft: Bedrock Edition Device/Version

1.19.0

**Comment by YHDiamond** [comment]
I was able to replicate this on the latest Geyser spigot and latest Floodgate with Paper 1.19. It seems that these changes made by the debug stick do not show up on bedrock, even after the bedrock client relogs.

**Comment by onebeastchris** [comment]
Unfortunately, we can't fix this for fences specifically - for those, unlike walls, the client seems to calculate where it should and should connect by itself.

Walls should work properly, since the server dictates the block state to send, but there are no such block states for fences :/

tldr; we can't do much about this - unless Bedrock suddenly adds blockstates for fences.

**Comment by onebeastchris** [comment]
Closing this issue in favor of it being documented on our current limitations page: https://wiki.geysermc.org/geyser/current-limitations/

**Comment by THEMPGUYAlt** [comment]
Check the new updates, maybe they added block states? And if not, fallback: *STRUCTURE V.O.I.D*

**Comment by onebeastchris** [comment]
They didn't add new blockstates for fences; nor does using structure voids help here?

**Comment by Julmik647** [comment]
While you can't directly copy this addon, you can check how [debug stick addon](https://github.com/vytdev/debug-stick) and its [UI variant](https://github.com/vytdev/debug-stick-ui) handles it.
